picard/track.py

```diff
@@ -368,6 +374,18 @@ def _convert_folksonomy_tags_to_genre(self):
             join_with=config.setting['join_genres']
         )
 
+    def add_folksonomy_tags(self):
+        config = get_config()
+        tags = Counter(self._folksonomy_tags)
+        tags += self.album._folksonomy_tags
+        self.metadata['_folksonomy_tags'] = self._genres_to_metadata(tags, join_with=config.setting['join_genres'])
+
+    def add_genres(self):
+        config = get_config()
+        _genres = Counter(self._genres)
+        _genres += self.album._genres
+        self.metadata['_genres'] = self._genres_to_metadata(_genres, join_with=config.setting['join_genres'])
```

Comment:
As discussed on the chat the two methods here can be simplified. The suggestion you made in the chat seemed fine, just also consider my above comment as there is I think no need to run the lists through the `_genres_to_metadata` filtering.

Also please prefix those methods with an underscore, as I would consider them private for use inside the  `Track` class only.

Reply:
Yep, I moved the redundant code to a new function and directly assigned the _folksonomy_tags and _genres
fixed in commit: d543ec0b1277b5e69111195cf19b604dcd789f8a